pkl-core/src/main/java/org/pkl/core/Platform.java

```diff
@@ -17,14 +17,24 @@
 
 import com.oracle.truffle.api.Truffle;
 import com.oracle.truffle.api.TruffleOptions;
-import java.util.Objects;
 import org.graalvm.home.Version;
 
 /**
  * Information about the Pkl release that the current program runs on. This class is the Java
  * equivalent of standard library module {@code pkl.platform}.
+ *
+ * @param language the language implementation of this platform.
+ * @param runtime the language runtime of this platform.
+ * @param virtualMachine the virtual machine of this platform.
+ * @param operatingSystem the operating system of this platform.
+ * @param processor the processor of this platform.
```

Comment:
Not the _most_ elucidating documentation ;) I'd leave it out, tbh (also makes it easier to spot it isn't _really_ documented yet)